cogment_verse/mlflow_experiment_tracker.py

```diff
@@ -84,7 +85,8 @@ def _get_mlflow_client(self):
                 self._mlflow_exp_id = client.create_experiment(mlflow_experiment_name)
 
         if not self._mlflow_run_id:
-            run = client.create_run(self._mlflow_exp_id, tags={MLFLOW_RUN_NAME: self._run_id})
+            # run = client.create_run(self._mlflow_exp_id, tags={MLFLOW_RUN_NAME: self._run_id})
+            run = client.create_run(self._mlflow_exp_id)
```

Comment:
humm, why?

Reply:
There will be some bugs in "mflow==1.30" if I dont remove the `tag`. The reason that we need  `mlflow==1.30` because it allows downloading the csv file for each individual trial trun

Reply:
Ok then correct the bug with that as-is we are losing the connection between the cogment verse run name and the mlflow run name which mean you can't really know what is what

Reply:
Done. Install newest version `2.2.2` solved the issue.